miden/tests/integration/air/memory.rs

```diff
@@ -18,18 +19,30 @@ fn pushw() {
 
 #[test]
 fn pop() {
-    let asm_op = "pop.mem.0";
-    let pub_inputs = vec![1];
+    let asm_op = "pop.mem.0 pop.mem.0 pop.mem.0";
```

Comment:
I'd prefer to keep op tests (the ones built with `build_op_test!`) to a single operation if possible. Maybe it makes sense to create a separate more complex test which would use `build_test!` macro?